### DISCUSSION TITLE
render horizontal svg

### DISCUSSION THREAD

**Discussion by gerardoscoppetta** [opensheetmusicdisplay/opensheetmusicdisplay#1545] [Q&A]
Is there a way to render a score horizontally into SVG by dividing it into pages instead of all vertically?
I've placed a demo below to help you understand what I would like to achieve.
Thank you
https://www.suonosubito.net/test_opensheetmusicdisplay/index002.php

**Comment by sschmidTU** [comment]
You can set a pageFormat via osmd.setOptions() (or in the constructor), then you might be able to re-arrange the page divs that are displayed vertically into a horizontal layout.

**Reply by sschmidTU** [reply]
There's another idea, it's probably bad, but I'll mention it anyways: You could also use several OSMD instances for the same piece and set `MinMeasureToDrawIndex` and `MaxMeasureToDrawIndex`, making them only draw from measure index x to measure index y. The problem here is that you don't know how many measures will fit into one of your 'pages' until you've tried it with a score. But it could work hardcoded for specific scores that you optimize it manually for.
But basically, pageFormat will do the reasonable arranging for you, just vertically instead of horizontally, so you could try to rearrange the layout after render().

**Reply by sschmidTU** [reply]
Going back to pageFormat, if you set e.g. `osmd.setOptions("pageFormat: A4 P")`, if the score is long enough to exceed a page, you will have multiple divs with ids `osmdCanvasPage1`, `osmdCanvasPage2`, etc. - also SVGs with id `osmdSvgPage1` etc.

**Reply by gerardoscoppetta** [reply]
Thanks for the precious support :)
I managed to get what I was looking for.
Here is the example at the link I add below.
https://suonosubito.net/test_opensheetmusicdisplay/index004.php
